libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts

```diff
@@ -251,6 +252,17 @@ function addSkills(services: ServiceCollection, configuration: Configuration): v
 }
 
 function addCoreBot(services: ServiceCollection, configuration: Configuration): void {
+    services.addFactory<BotFrameworkAuthentication, { authenticationConfiguration: AuthenticationConfiguration }>(
+        'botFrameworkAuthentication',
+        ['authenticationConfiguration'],
+        (dependencies) =>
+            createBotFrameworkAuthenticationFromConfiguration(
+                configuration,
+                undefined,
```

Comment:
@stevengum, not sure this is right. Thoughts?

Reply:
Right, if there a `ServiceClientCredentialFactory` keyed under `'serviceClientCredentialFactory'` already available, this `addFactory()` call should use that first. Otherwise pass in `undefined` and rely on `ConfigurationBotFrameworkAuthentication` to create a `ServiceClientCredentialFactory` in its constructor.